core-services/prow/02_config/openshift/dpu-network-operator/_pluginconfig.yaml

```diff
@@ -11,3 +11,20 @@ plugins:
   openshift/dpu-network-operator:
     plugins:
     - approve
+label:
+  restricted_labels:
+    openshift/dpu-network-operator:
+    - allowed_users:
+      - zshi-redhat
+      - pliurh
```

Comment:
Can we only add the team name here? I think both @zshi-redhat and me are in the team-multus-cni-maintainers

Reply:
yes,  updated.  just copy from sriov operator :)